internal/database/postgres_migrate.go

```diff
@@ -88,6 +88,37 @@ CREATE TABLE irc_channel
     UNIQUE (network_id, name)
 );
 
+CREATE TABLE release_profile_duplicate
+(
+    id            SERIAL PRIMARY KEY,
+    name          TEXT NOT NULL,
+    protocol      BOOLEAN DEFAULT FALSE,
+    release_name  BOOLEAN DEFAULT FALSE,
+    title         BOOLEAN DEFAULT FALSE,
+    year          BOOLEAN DEFAULT FALSE,
+    month         BOOLEAN DEFAULT FALSE,
+    day           BOOLEAN DEFAULT FALSE,
+    source        BOOLEAN DEFAULT FALSE,
+    resolution    BOOLEAN DEFAULT FALSE,
+    codec         BOOLEAN DEFAULT FALSE,
+    container     BOOLEAN DEFAULT FALSE,
+    hdr           BOOLEAN DEFAULT FALSE,
+    audio         BOOLEAN DEFAULT FALSE,
+    release_group BOOLEAN DEFAULT FALSE,
+    season        BOOLEAN DEFAULT FALSE,
+    episode       BOOLEAN DEFAULT FALSE,
+    website       BOOLEAN DEFAULT FALSE,
+    proper        BOOLEAN DEFAULT FALSE,
+    repack        BOOLEAN DEFAULT FALSE
+);
+
+INSERT INTO release_profile_duplicate (id, name, protocol, release_name, title, year, month, day, source, resolution, codec, container, hdr, audio, release_group, season, episode, website, proper, repack)
+VALUES (1, 'Exact release', 'f', 't', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f'),
+       (2, 'Movie', 'f', 'f', 't', 't', 'f', 'f', 'f', 't', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f'),
+       (3, 'TV', 'f', 'f', 't', 't', 't', 't', 'f', 't', 'f', 'f', 'f', 'f', 'f', 't', 't', 'f', 'f', 'f'),
+       (4, 'Daily TV show', 'f', 'f', 't', 't', 't', 't', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f'),
+       (5, 'Episodic TV show', 'f', 'f', 't', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 't', 't', 'f', 'f', 'f');
```

Comment:
Pretty sure PG also supports 1 0 as true/false, could be good to have them the same, but not critical.

Reply:
I agree with the sentiment and I started that way, but no, it doesn't.
```
evergreen=# create temp table t(b bool);
CREATE TABLE
evergreen=# insert into t (b) values (1);
ERROR:  column "b" is of type boolean but expression is of type integer
evergreen=# insert into t (b) values ('t');
INSERT 0 1
evergreen=# insert into t (b) values (true);
INSERT 0 1```